frontend/catalyst/compiler.py

```diff
@@ -157,7 +157,7 @@ class BufferizationPass(PassPipeline):
 
     _executable = get_executable_path("quantum", "quantum-opt")
     _default_flags = [
-        "--inline",
+        # "--inline",
```

Comment:
I think inlining can be reinstated, if you sync with main the issues should be resolved after #121 was merged. (cse and buffer-deallocation were disabled)

Reply:
OK, I need to check it against the 121

Reply:
I have reverted this change, rebased the PR and run the memory issue testing example several times. I can see no problems now, thanks!